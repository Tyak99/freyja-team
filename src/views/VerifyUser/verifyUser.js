import React from 'react';
import VerifyUser from '../../components/VerifyUser/VerifyUser';

const VerifyUserPage = () => <VerifyUser />;

export default VerifyUserPage;
