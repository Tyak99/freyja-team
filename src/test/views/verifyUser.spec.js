import React from 'react';
import { shallow } from 'enzyme';
import VerifyUser from '../../views/VerifyUser/verifyUser';

describe('test verify user page', () => {
  const verify = shallow(<VerifyUser />);

  it('should test verify user page', () => {
    expect(verify).toMatchSnapshot();
  });
});
