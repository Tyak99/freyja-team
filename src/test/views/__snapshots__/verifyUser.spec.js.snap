// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test verify user page should test verify user page 1`] = `ShallowWrapper {}`;
